test/test_mutable_binheap.jl

```diff
@@ -121,6 +126,7 @@ end
     end
 
     @testset "hmin / push! / pop!" begin
+        vs = [4, 1, 3, 2, 16, 9, 10, 14, 8, 7]
```

Comment:
this is a good change anyway. Testsets hould be indipendent.
Though maybe we move this down to after `ss` is declared?

Reply:
Yes, they both are valid heaps. I also confirmed it using this function [here](https://www.geeksforgeeks.org/how-to-check-if-a-given-array-represents-a-binary-heap/#:~:text=An%20Efficient%20Solution%20is%20to,%2C%20then%20x%20%3E%20z).